specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-10-01/containerInstance.json

```diff
@@ -1585,6 +1651,111 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "CachedImagesListResult": {
+      "description": "The response containing cached images.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/cachedImages"
+          },
+          "description": "The list of cached images."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of cached images."
+        }
+      }
+    },
+    "cachedImages": {
```

Comment:
does it make sense for these to have the ARM resource Id?

Reply:
Is there a future scenario for these to be shared and reused? Then what happens to RBAC?

Reply:
it's right now a public, relatively static list. Anyone have permission to the subscription should be able to read the list.

Reply:
I'd say it's not necessary right now, but if really needed, adding an Id is not a problem.

Reply:
ok, will sign off on the manifest. lets add the id.